components/match2/commons/match_group_input.lua

```diff
@@ -52,6 +52,7 @@ function MatchGroupInput.readMatchlist(bracketId, args)
 		bracketData.header = args['M' .. matchIndex .. 'header'] or bracketData.header
 		bracketData.bracketindex = Variables.varDefault('match2bracketindex', 0)
 		bracketData.sectionheader = sectionHeader
+		bracketData.dateheader = Logic.readBool(match.dateheader)
```

Comment:
Could consider omitting `dateheader` if false in the lpdb record. It reduces clutter a bit at the expense of being less explicit. 

```
bracketData.dateheader = Logic.readBool(match.dateheader) or nil
```